modules/logging-loki-storage-minio.adoc

```diff
@@ -1,22 +1,21 @@
 // Module is included in the following assemblies:
-// logging/cluster-logging-loki.adoc
+// logging/log_storage/installing-log-storage.adoc
 //
 :_mod-docs-content-type: PROCEDURE
 [id="logging-loki-storage-minio_{context}"]
 = Minio storage
```

Comment:
Procedure titles should begin with a gerund, such as "Configuring Minio storage".

Reply:
Out of scope